javascript/photo.js

```diff
@@ -129,14 +129,15 @@ backgrounds.Photo = new Model({
 
     /**
      * Displays the image.
+     * TODO: Refactor this function to make it more readable.
      */
     display: function(limitImages, cacheSize) {
         this.hideShareButtons();
         count = localStorage.getItem("count");
         if (count === null) {
             this.loadFirstImage();
-        } 
-        else if (Number(count) < 3) {
+        }
+        else if (Number(count) < this.CNT_BACKUP_IMAGES) {
```

Comment:
Let's not change the number of images used at startup to be the same as the number of backup images. Currently, users will see only 3 of the 5 backup images at startup. Let's leave it that way to preserve the illusion of randomness.

Reply:
In fact ideally, let's only get backup images if it fails to pull new images?

Reply:
The current version (without this commit) already does all of that.  This commit assumes that the usage of 3 instead of 5 is a bug and not a feature.  But the randomness logic sounds good to me so... will just drop this commit.